src/wasm-stack.h

```diff
@@ -133,6 +135,7 @@ class BinaryInstWriter : public OverriddenVisitor<BinaryInstWriter> {
   Function* func = nullptr;
   bool sourceMap;
   bool DWARF;
+  bool Memory64;
```

Comment:
For consistency, better to use `memory64`

Reply:
address comment, thanks